lightclient/src/background.rs

```diff
@@ -176,10 +177,15 @@ impl<TPlatform: PlatformRef, TChain> BackgroundTask<TPlatform, TChain> {
                         tracing::trace!(target: LOG_TARGET, "Smoldot RPC responses channel closed");
                         break;
                     };
+                    let (maybe_truncated, delim) = if back_message.len() > MAX_LOG_SIZE {
+                        (&back_message[0..MAX_LOG_SIZE], "...")
+                    } else {
+                        (&back_message[..], "")
+                    };
```

Comment:
Do we need to worry about invalid UTF boundaries here too?

(perhaps a little helper fn which trims a message to keep it under some max length and uses `is_char_boundary` internally would be ideal?)